.github/workflows/build-linux-arm64-installer.yml

```diff
@@ -105,13 +105,24 @@ jobs:
         sha256sum ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb > ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.sha256
         ls ${{ github.workspace }}/build_scripts/final_installer/
 
+    - name: Install py3createtorrent
+      if: startsWith(github.ref, 'refs/tags/')
+      run: |
+        pip3 install py3createtorrent
+
+   - name: Create torrent
+     if: startsWith(github.ref, 'refs/tags/')
+     run: |
+       py3createtorrent -t torrent.chia.net ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb ${{ github.workspace }}/build_scripts/final_installer/chia-blockchain_${{ steps.version_number.outputs.CHIA_INSTALLER_VERSION }}_arm64.deb.torrent
```

Comment:
I would imagine we'd also want to add the URL where we host this file too, as a `--webseed`

Reply:
Yeah, this was actually just a test run, this part of the automation is going to be moved to a different project.